tests/sentry/types/test_region.py

```diff
@@ -49,5 +49,10 @@ def test_get_local_region(self):
                 assert get_local_region() == regions[0]
 
             with override_settings(SILO_MODE=SiloMode.MONOLITH):
-                with pytest.raises(RegionContextError):
-                    get_local_region()
+                # The relative address and the 0 id are the only important parts of this region value
```

Comment:
Not quite -- in many cases we store just the name of a region, so it's important we pick a name that clearly distinguishes from "true" regions in the future.  The name should be consistent in any other place we need to "replicate" within monolith (essentially run local outboxes).